doc/02-Installation.md

```diff
@@ -198,10 +218,14 @@ systemctl start icingadb
 
 ### Enable remote Redis connections <a id="remote-redis"></a>
 
-By default `icingadb-redis` listens only on `127.0.0.1`. If you want to change that (e.g. Icinga Web 2), just change `bind 127.0.0.1 ::1` and `protected-mode yes` in `/etc/icinga-redis/icingadb-redis.conf` to the interface you want to use and `protected-mode no`.
+By default `icingadb-redis` listens only on `127.0.0.1`. If you want to change that (e.g. Icinga Web 2), just
```

Comment:
I'd be a bit more verbose: "e.g. for Icinga Web 2 running on another machine" or something like that.